lib/screens/Privacy/change_password.dart

```diff
@@ -0,0 +1,171 @@
+import 'package:flutter/material.dart';
+
+class PasswordScreen extends StatefulWidget {
+  @override
+  _PasswordState createState() => _PasswordState();
+}
+
+class _PasswordState extends State<PasswordScreen> {
+  @override
+  Widget build(BuildContext context) {
+    int _selectedIndex = 0;
+    String _text = 'BlaBlaBla';
+    void _onItemTapped(int index) {
+      setState(() {
+        _selectedIndex = index;
+      });
+      if (index == 1) {
+        Navigator.of(context).pushNamed('CameraPage');
+      } else if (index == 0) {
+        Navigator.of(context).pushNamed('DocumentPage');
+      }
+    }
```

Comment:
I noticed a lot of similar code to this. Is there a reason for it being in multiple files? Ie, how does it affect navigation etc?